ccdb/index.md

```diff
@@ -6,33 +6,46 @@ layout: ccdb
 
 ### What is this data?
 
-Each week we send thousands of consumers' complaints about financial products and services to companies for response. Complaints are listed in the database after the company responds or after they’ve had the complaint for 15 calendar days, whichever comes first. We publish the consumer’s description of what happened if the consumer opts to share it and after taking steps to remove personal information. See our 
+Each week we send thousands of consumers' complaints about financial products and services to companies for response. Complaints are listed in the database after the company responds or after they’ve had the complaint for 15 calendar days, whichever comes first. We publish the consumer’s description of what happened if the consumer opts to share it and after taking steps to remove personal information. See our
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/a/assets/201503_cfpb_Narrative-Scrubbing-Standard.pdf">
     <span class="icon-link_text">Scrubbing Standard</span>
-</a> and 
+</a> and
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/201509_cfpb_evaluation-of-narrative-scrubbing-standard-and-process.pdf">
     <span class="icon-link_text">Third Party Study</span>
 </a> for more details.
 
-We don’t verify all the facts alleged in these complaints, but we take steps to confirm a commercial relationship between the consumer and the company. We may remove complaints if they don’t meet all of the publication criteria as defined in our 
+We don’t verify all the facts alleged in these complaints, but we take steps to confirm a commercial relationship between the consumer and the company. We may remove complaints if they don’t meet all of the publication criteria as defined in our
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/201303_cfpb_Final-Policy-Statement-Disclosure-of-Consumer-Complaint-Data.pdf">
     <span class="icon-link_text">policy statement</span>
-</a>. Data is generally refreshed nightly. Company level information should be considered in context of company size and/or market share.
+</a>. Data is generally refreshed daily. Company level information should be considered in context of company size and/or market share.
 
-The Consumer Complaint Database contains complaints we've received about consumer financial products and services, including:
+The Consumer Complaint Database contains complaints we’ve received about consumer financial products and services. In April 2017 we updated the form consumers use to submit complaints. The changes include reorganizing how products, sub-products, issues, and sub-issues are grouped and making some plain language improvements.
```

Comment:
Stakeholders are reviewing this suggestion now. If everyone signs off on it before the scheduled launch, I'll make this fix. Otherwise, we can add it as a follow-up fix.